packages/terra-clinical-header/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Unreleased
 
 ### Changed
```

Comment:
This isn't a result of this pull request, but here are two unique changed sections now under unreleased. Would you mind cleaning them up?

Reply:
updated https://github.com/cerner/terra-clinical/pull/529/commits/3977fc628ed9333cc59b80ad72038ce2c500b58d